homebrew_challenge/src/components/Utils/RssFeedUtils.test.js

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
These changes are also in PR 35

Reply:
You'll want to merge master into this branch now that PR 35 is merged in. There may be conflicts, but the end result will be cleaner